test/e2e/agent/config_test.go

```diff
@@ -7,6 +7,9 @@ package agent
 import (
 	"testing"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
```

Comment:
nit: imports formatting

Reply:
I thought this is the right formatting (three groups) 🙂 Changing to two.